manifests/crds/applicationset-crd.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   labels:
-    app.kubernetes.io/name: applicationsets.argoproj.io
-  name: applicationsets.argoproj.io
+    app.kubernetes.io/name: applicationsets.appset.argoproj.io
+  name: applicationsets.appset.argoproj.io
 spec:
-  group: argoproj.io
+  group: appset.argoproj.io
```

Comment:
I'm pretty sure we don't want this to happen. It would require all the users to update their ApplicationSet manifests. I know the other solution was moving a bunch of files around. Want to give that a go?